### PR TITLE
net::Defaults: Revert and re-apply revised code

### PR DESCRIPTION
**Pull request by sgothel**
* Resolves: #9833  refining PR #9916, partially replacing PR #10355
* Target version: master 

### Summary
net::Defaults was a singleton, becoming a global, keeping untouched constexpr in SocketPoll.

- Reverts commit 4499e42b762ef3f96fc20a377cf7d663b48305d2
- SocketPoll::DefaultPollTimeoutMicroS is maintained original
- net::Defaults is a global instance w/ default values
  - inactivityTimeout 3600s
    - instead of using DefaultPollTimeoutMicroS for {idle->inactivity) checkRemoval
  - wsPingAvgTimeout 12s
  - wsPingInterval 18s
  - maxTCPConnections 200000 (arbitrary value)
      - for limiting connections in SocketPoll::poll,
        instead of using the session count

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: net/Socket.cpp [hunk 1]
```diff
@@ -51,16 +51,15 @@
 #include <openssl/x509v3.h>
 #endif
 #include "WebSocketHandler.hpp"
-#include <net/HttpRequest.hpp>
-#include <NetUtil.hpp>
```
Comment (caolanm) [inline net/Socket.cpp:55]: I wonder why change the include style as part of this?

Comment (sgothel) [reply net/Socket.cpp:55]: yes .. and I remember that the system include style was not recommended for project includes. not thoroughly applied.

Comment (sgothel) [reply net/Socket.cpp:55]:
```
-#include <net/HttpRequest.hpp>
-#include <NetUtil.hpp>
+#include "net/HttpRequest.hpp"
+#include "net/NetUtil.hpp"
```

**Review by mmeeks — APPROVED** [review]
Thanks.

**Review by ravin00 — APPROVED** [review]

**Comment by sgothel** [comment]
Cypress desktop failures:
- writer/invalidations_spec.js
- writer/searchbar_spec.js

**Review by ravin00 — COMMENTED** [review]
If all the changes are done accordingly you can merge this into the main branch

**Pull request merged**